speaking-portal-project/src/phonemeFactory/rhubarb.ts

```diff
@@ -35,26 +35,23 @@ export async function rhubarbProcessor(
         '-d',
         `.${text_file_name}`,
         '--extendedShapes',
-        'GX',
+        'GHX',
```

Comment:
I was also just about to make this change! thanks for doing this!

Reply:
No problemo!